ipaserver/install/ipactl.py

```diff
@@ -567,7 +669,12 @@ def main():
     if len(args) != 1:
         # LSB status code 2: invalid or excess argument(s)
         raise IpactlError("You must specify one action", 2)
-    elif args[0] != "start" and args[0] != "stop" and args[0] != "restart" and args[0] != "status":
+    elif (
+        args[0] != "start" and
+        args[0] != "stop" and
+        args[0] != "restart" and
+        args[0] != "status"
+    ):
```

Comment:
Since you're fixing things up why not change this to:
elif args[0] not in ("start", "stop", "restart", "status"):

Reply:
Agreed, doin' so.